src/data_cell/graph_datacell.h

```diff
@@ -138,6 +141,18 @@ GraphDataCell<IOTmpl, false>::GetNeighbors(InnerIdType id,
         neighbor_ids.size() * sizeof(InnerIdType), start, (uint8_t*)(neighbor_ids.data()));
 }
 
+template <typename IOTmpl>
+void
+GraphDataCell<IOTmpl, false>::Resize(InnerIdType new_size) {
+    if (new_size < this->max_capacity_) {
+        return;
+    }
+    this->max_capacity_ = new_size;
+    uint64_t io_size = new_size * code_line_size_;
+    uint8_t end_flag = 127;
```

Comment:
Defined as static and constant.

Reply:
actually the value of this is meaningless, I think add a comment is useful